js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,15 @@
+function sumAll(numbers) {
+  let num1 = numbers[0];
+  let num2 = numbers[1];
```

Comment:
You could also use object destructuring here
```suggestion
function sumAll([num1, num2]) {
```